### ISSUE TITLE
(Have an option to) store cache in ~/.cache instead of ~/.modrana/...

### ISSUE THREAD

**Issue by radomirbosak** [M4rtinK/modrana#203]

**Comment by M4rtinK** [comment]
Good point - we should really use the recommended XDG path for the profile on new installations. But at the same time we should not break user installations using the old path. But that should be doable, by checking if the old path exists before creating & using the new path.
